### ISSUE TITLE
void-variable orel-mod

### ISSUE THREAD

**Issue by fleimgruber** [jkitchin/org-ref#826]
Just tried the new version from MELPA (org-ref-20201230.2020) and with the new `org-ref` command (used in the example file in https://github.com/jkitchin/org-ref/issues/787#issuecomment-752376559) I get the traceback at the end. AFAICS, git blamed it on 734754f8b836ce834233e8eaae1c40cfc3db8091.

```
Debugger entered--Lisp error: (void-variable orel-mod)
  (and orel-mod orelc-mod)
  (if (and orel-mod orelc-mod) (progn (time-less-p orel-mod orelc-mod) (insert "- org-ref.elc is older than org-ref.el. That is pr...") (insert "- load-prefer-newer = %s\n" load-prefer-newer) (insert "  consider (setq load-prefer-newer t) or using htt...")))
  (if (let* ((org-ref-el (concat (file-name-sans-extension (locate-library "org-ref")) ".el")) (orel-mod) (org-ref-elc (concat (file-name-sans-extension (locate-library "org-ref")) ".elc")) (orelc-mod) (elc-version)) (if (file-exists-p org-ref-el) (progn (setq orel-mod (file-attribute-modification-time (file-attributes org-ref-el))))) (if (file-exists-p org-ref-elc) (progn (setq orelc-mod (file-attribute-modification-time (file-attributes org-ref-elc)))))) nil (if (and orel-mod orelc-mod) (progn (time-less-p orel-mod orelc-mod) (insert "- org-ref.elc is older than org-ref.el. That is pr...") (insert "- load-prefer-newer = %s\n" load-prefer-newer) (insert "  consider (setq load-prefer-newer t) or using htt..."))) (if (and org-ref-elc (file-exists-p org-ref-elc)) (progn (setq elc-version (let ((temp-buffer (generate-new-buffer " *temp*"))) (save-current-buffer (set-buffer temp-buffer) (unwind-protect (progn ... ... ... ...) (and ... ...)))) (if (string= elc-version (format "%s.%s" emacs-major-version emacs-minor-version)) nil (insert "- %s compiled with Emacs %s but you are running %s..." elc-version emacs-major-version emacs-minor-version))))))
  (save-current-buffer (set-buffer buf) (read-only-mode -1) (erase-buffer) (org-mode) (insert (format "#+title: org-ref report on [[%s][%s]]\n\n" (buffer-file-name cb) (buffer-name cb))) (if bad-citations (progn (insert "* Bad citations\n") (let* ((--cl-var-- bad-citations) (marker nil) (key nil)) (while (consp --cl-var--) (setq marker (car --cl-var--) key (car-safe (prog1 marker ...))) (setq mbuffer (buffer-name (marker-buffer marker)) mchar (marker-position marker)) (insert (format "- [[elisp:(progn (switch-to-buffer %S) (goto-char ..." mbuffer mchar key)) (setq --cl-var-- (cdr --cl-var--))) nil))) (if bad-refs (progn (insert "\n* Bad ref links\n") (let* ((--cl-var-- bad-refs) (marker nil) (key nil)) (while (consp --cl-var--) (setq marker (car --cl-var--) key (car-safe (prog1 marker ...))) (setq mbuffer (buffer-name (marker-buffer marker)) mchar (marker-position marker)) (insert (format "- [[elisp:(progn (switch-to-buffer %S) (goto-char ..." mbuffer mchar key)) (setq --cl-var-- (cdr --cl-var--))) nil))) (if bad-labels (progn (insert "\n* Multiply defined label links\n") (let* ((--cl-var-- bad-labels) (marker nil) (key nil)) (while (consp --cl-var--) (setq marker (car --cl-var--) key (car-safe (prog1 marker ...))) (setq mbuffer (buffer-name (marker-buffer marker)) mchar (marker-position marker)) (insert (format "- [[elisp:(progn (switch-to-buffer %S) (goto-char ..." mbuffer mchar key)) (setq --cl-var-- (cdr --cl-var--))) nil))) (if unreferenced-labels (progn (insert "\n* Unreferenced label links\n") (let* ((--cl-var-- unreferenced-labels) (marker nil) (key nil)) (while (consp --cl-var--) (setq marker (car --cl-var--) key (car-safe (prog1 marker ...))) (if (not (string= key "")) (progn (setq mbuffer ... mchar ...) (insert ...))) (setq --cl-var-- (cdr --cl-var--))) nil))) (if bib-candidates (progn (insert "\n* Bibliography\n") (let* ((--cl-var-- bib-candidates) (candidate nil)) (while (consp --cl-var--) (setq candidate (car --cl-var--)) (insert (format "- %s\n" candidate)) (setq --cl-var-- (cdr --cl-var--))) nil))) (insert "\n* Miscellaneous\n") (let* ((--cl-var-- (list (format "org-latex-prefer-user-labels = %s" org-latex-prefer-user-labels) (format "bibtex-dialect = %s" bibtex-dialect) (format "biblatex is%srequired." (if biblatex-required " " " not ")) (format "biblatex is%sused." (if biblatex-used " " " not ")) (format "emacs-version = %s" (emacs-version)) (format "org-version = %s" (org-version)) (org-ref-version) (format "completion backend = %s" org-ref-completion-library) (format "org-latex-pdf-process is defined as %s" org-latex-pdf-process) (format "natbib is%srequired." (if natbib-required " " " not ")) (format "natbib is%sin org-latex-default-packages-alist or ..." (if natbib-used " " " not ")) (format "cleveref is%srequired." (if cleveref-required " " " not ")) (format "cleveref is%sin org-latex-default-packages-alist o..." (if cleveref-used " " " not ")))) (s nil)) (while (consp --cl-var--) (setq s (car --cl-var--)) (insert "- " s "\n") (setq --cl-var-- (cdr --cl-var--))) nil) (insert (format "- org-latex-default-packages-alist\n")) (let* ((--cl-var-- org-latex-default-packages-alist) (el nil)) (while (consp --cl-var--) (setq el (car --cl-var--)) (insert (format "  %S\n" el)) (setq --cl-var-- (cdr --cl-var--))) nil) (if (null org-latex-packages-alist) (insert "-  org-latex-packages-alist is nil\n") (insert "-  org-latex-packages-alist\n") (let* ((--cl-var-- org-latex-packages-alist) (el nil)) (while (consp --cl-var--) (setq el (car --cl-var--)) (insert (format "  %S\n" el)) (setq --cl-var-- (cdr --cl-var--))) nil)) (if (let* ((org-ref-el (concat (file-name-sans-extension (locate-library "org-ref")) ".el")) (orel-mod) (org-ref-elc (concat (file-name-sans-extension (locate-library "org-ref")) ".elc")) (orelc-mod) (elc-version)) (if (file-exists-p org-ref-el) (progn (setq orel-mod (file-attribute-modification-time (file-attributes org-ref-el))))) (if (file-exists-p org-ref-elc) (progn (setq orelc-mod (file-attribute-modification-time (file-attributes org-ref-elc)))))) nil (if (and orel-mod orelc-mod) (progn (time-less-p orel-mod orelc-mod) (insert "- org-ref.elc is older than org-ref.el. That is pr...") (insert "- load-prefer-newer = %s\n" load-prefer-newer) (insert "  consider (setq load-prefer-newer t) or using htt..."))) (if (and org-ref-elc (file-exists-p org-ref-elc)) (progn (setq elc-version (let ((temp-buffer ...)) (save-current-buffer (set-buffer temp-buffer) (unwind-protect ... ...))) (if (string= elc-version (format "%s.%s" emacs-major-version emacs-minor-version)) nil (insert "- %s compiled with Emacs %s but you are running %s..." elc-version emacs-major-version emacs-minor-version)))))) (insert (format "\n* Utilities\n\n- [[elisp:(progn (find-file %S) (isp..." fname fname)) (goto-char (point-min)) (setq header-line-format "Press q to quit.") (local-set-key "q" #'(lambda nil (interactive) (delete-window))) (read-only-mode))
  (let* ((buf (get-buffer-create "*org-ref*")) (cb (current-buffer)) (fname (buffer-file-name)) (bad-citations (org-ref-bad-cite-candidates)) (bad-refs (org-ref-bad-ref-candidates)) (bad-labels (org-ref-bad-label-candidates)) (bad-files (org-ref-bad-file-link-candidates)) (bib-candidates 'nil) (unreferenced-labels 'nil) natbib-required natbib-used cleveref-required cleveref-used biblatex-required biblatex-used mbuffer mchar (org-latex-prefer-user-labels (and (boundp ...) org-latex-prefer-user-labels))) (org-element-map (org-element-parse-buffer) 'link #'(lambda (link) (if ... ...) (if ... ...) (if ... ...)) nil t) (setq natbib-used (or (member "natbib" (mapcar ... org-latex-default-packages-alist)) (member "natbib" (mapcar ... org-latex-packages-alist)) (save-excursion (goto-char ...) (re-search-forward "{natbib}" nil t)))) (setq biblatex-used (or (member "biblatex" (mapcar ... org-latex-default-packages-alist)) (member "biblatex" (mapcar ... org-latex-packages-alist)) (save-excursion (goto-char ...) (re-search-forward "{biblatex}" nil t)))) (setq cleveref-used (or (member "cleveref" (mapcar ... org-latex-default-packages-alist)) (member "cleveref" (mapcar ... org-latex-packages-alist)) (save-excursion (goto-char ...) (re-search-forward "{cleveref}" nil t)))) (let* ((--cl-var-- (org-ref-find-bibliography)) (bibfile nil)) (while (consp --cl-var--) (setq bibfile (car --cl-var--)) (let (...) (save-current-buffer ... ...) (let* ... ...)) (setq --cl-var-- (cdr --cl-var--))) nil) (save-excursion (goto-char 0) (if (re-search-forward "bibliographystyle:\\|\\\\bibliograp..." nil t) nil (if (memql "No bibliography style found. Thi..." bib-candidates) (with-no-warnings bib-candidates) (setq bib-candidates ...)))) (save-excursion (goto-char 0) (if (re-search-forward "bibliographystyle:" nil t) (progn (let ... ...)))) (let* ((bib-links (-filter ... ...)) (n-bib-links (length bib-links))) (if (> n-bib-links 1) (progn (mapc ... bib-links)))) (mapc #'(lambda (bibfile) (if ... nil ...)) (org-ref-find-bibliography)) (let ((bibfiles (mapcar ... ...))) (mapc #'(lambda ... ...) bibfiles)) (let ((bibfiles (mapcar ... ...))) (mapc #'(lambda ... ...) bibfiles) (mapc #'(lambda ... ...) bibfiles)) (save-excursion (save-restriction (widen) (goto-char (point-min)) (let (...) (while ... ...) (save-excursion ... ...) (save-excursion ... ...) (let* ... ... nil) (org-map-entries ...) (goto-char ...) (while ... ...) (let ... ...)))) (save-current-buffer (set-buffer buf) (read-only-mode -1) (erase-buffer) (org-mode) (insert (format "#+title: org-ref report on [[%s]..." (buffer-file-name cb) (buffer-name cb))) (if bad-citations (progn (insert "* Bad citations\n") (let* ... ... nil))) (if bad-refs (progn (insert "\n* Bad ref links\n") (let* ... ... nil))) (if bad-labels (progn (insert "\n* Multiply defined label links\n") (let* ... ... nil))) (if unreferenced-labels (progn (insert "\n* Unreferenced label links\n") (let* ... ... nil))) (if bib-candidates (progn (insert "\n* Bibliography\n") (let* ... ... nil))) (insert "\n* Miscellaneous\n") (let* ((--cl-var-- ...) (s nil)) (while (consp --cl-var--) (setq s ...) (insert "- " s "\n") (setq --cl-var-- ...)) nil) (insert (format "- org-latex-default-packages-ali...")) (let* ((--cl-var-- org-latex-default-packages-alist) (el nil)) (while (consp --cl-var--) (setq el ...) (insert ...) (setq --cl-var-- ...)) nil) (if (null org-latex-packages-alist) (insert "-  org-latex-packages-alist is n...") (insert "-  org-latex-packages-alist\n") (let* (... ...) (while ... ... ... ...) nil)) (if (let* (... ... ... ... ...) (if ... ...) (if ... ...)) nil (if (and orel-mod orelc-mod) (progn ... ... ... ...)) (if (and org-ref-elc ...) (progn ...))) (insert (format "\n* Utilities\n\n- [[elisp:(progn (..." fname fname)) (goto-char (point-min)) (setq header-line-format "Press q to quit.") (local-set-key "q" #'(lambda nil ... ...)) (read-only-mode)) (display-buffer-in-side-window buf '((side . right))))
  org-ref()
  funcall-interactively(org-ref)
  call-interactively(org-ref record nil)
  command-execute(org-ref record)
  helm-M-x-execute-command(org-ref)
  helm-execute-selection-action-1()
  helm-execute-selection-action()
  helm-internal((((name . "Emacs Commands history") (candidates . #f(compiled-function () #<bytecode 0x1ff709a71845>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keep...") (must-match . t) (group . helm-command) (match-dynamic . t)) ((name . "Emacs Commands") (candidates . #f(compiled-function () #<bytecode 0x1ff70d103ccd>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keep...") (must-match . t) (group . helm-command) (match-dynamic . t))) nil "M-x " nil nil "*helm M-x*" nil nil helm-M-x-input-history)
  apply(helm-internal ((((name . "Emacs Commands history") (candidates . #f(compiled-function () #<bytecode 0x1ff709a71845>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) (8 . helm-next-source) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ...) (67108987 . helm-enlarge-window) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keeping se...") (must-match . t) (group . helm-command) (match-dynamic . t)) ((name . "Emacs Commands") (candidates . #f(compiled-function () #<bytecode 0x1ff70d103ccd>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) (8 . helm-next-source) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ...) (67108987 . helm-enlarge-window) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keeping se...") (must-match . t) (group . helm-command) (match-dynamic . t))) nil "M-x " nil nil "*helm M-x*" nil nil helm-M-x-input-history))
  helm((((name . "Emacs Commands history") (candidates . #f(compiled-function () #<bytecode 0x1ff709a71845>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keep...") (must-match . t) (group . helm-command) (match-dynamic . t)) ((name . "Emacs Commands") (candidates . #f(compiled-function () #<bytecode 0x1ff70d103ccd>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keep...") (must-match . t) (group . helm-command) (match-dynamic . t))) nil "M-x " nil nil "*helm M-x*" nil nil helm-M-x-input-history)
  apply(helm ((((name . "Emacs Commands history") (candidates . #f(compiled-function () #<bytecode 0x1ff709a71845>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) (8 . helm-next-source) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ...) (67108987 . helm-enlarge-window) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keeping se...") (must-match . t) (group . helm-command) (match-dynamic . t)) ((name . "Emacs Commands") (candidates . #f(compiled-function () #<bytecode 0x1ff70d103ccd>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) (8 . helm-next-source) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ...) (67108987 . helm-enlarge-window) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keeping se...") (must-match . t) (group . helm-command) (match-dynamic . t))) nil "M-x " nil nil "*helm M-x*" nil nil helm-M-x-input-history))
  helm(:sources (((name . "Emacs Commands history") (candidates . #f(compiled-function () #<bytecode 0x1ff709a71845>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keep...") (must-match . t) (group . helm-command) (match-dynamic . t)) ((name . "Emacs Commands") (candidates . #f(compiled-function () #<bytecode 0x1ff70d103ccd>)) (keymap keymap (keymap ...) keymap (21 . helm-M-x-universal-argument) keymap (127 . delete-backward-char) (27 keymap ...) (C-return . helm-cr-empty-string) keymap (26 . helm-select-action) (tab . helm-execute-persistent-action) (33554440 . describe-key) (33554443 . helm-follow-action-backward) (33554442 . helm-follow-action-forward) (17 . ace-jump-helm-line) (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff70d7773b5>)) (f1 lambda nil ... ...) ...) (action . helm-type-command-actions) (persistent-action . helm-M-x-persistent-action) (persistent-help . "Describe this command") (help-message . helm-M-x-help-message) (requires-pattern . 0) (filtered-candidate-transformer helm-M-x-transformer-no-sort helm-flx-fuzzy-matching-sort helm-fuzzy-highlight-matches) (volatile . t) (match . identity) (redisplay . identity) (nomark . t) (coerce . helm-symbolify) (header-line . "TAB: Describe this command (keep...") (must-match . t) (group . helm-command) (match-dynamic . t))) :prompt "M-x " :buffer "*helm M-x*" :history helm-M-x-input-history)
  helm-M-x-read-extended-command([icomplete-fido-delete-char LaTeX-length-list diary-included-files eval-sexp-fu-flash-error org-babel-C-utility-header-to-C copy-epl-upgrade package-lint--sane-prefixes vc-git-stash-snapshot TeX-command-Biber edebug-trace js-jsx--text-properties ispell-menu-map-needed octave-close-block gnus-sync ranger-wdired LaTeX-item-tabular* faceup-get-text-properties ols none-but-delete \\overline-14 js--expression-in-sgml-indent-line sgml-syntax-propertize gdb-frame-locals-buffer delphi-column-of cl-print--preprocess org-publish-sanitize-plist eieio-class-tag--ede-target edebug-clear-frequency-count c-standard-font-lock-fontify-region-function bibtex-SuppCollection 0 TeX-auto-local firstname evil-paredit-yank-line c-mode-var \\prime Edebug\ All\ Defs org-duration-format helm-source-info-ess-defs \' nnoo-import-1 spacemacs/layouts-transient-state/spacemacs/layout-switch-to-and-exit org-ref-notes-directory rx-trans-forms ebrowse-ms-definition-pattern--cmacro tramp-gvfs-unmount \. icomplete--category pkg-info--read-function \\bullet ...])
  helm-M-x(nil)
  funcall-interactively(helm-M-x nil)
  call-interactively(helm-M-x)
  (let ((completion-styles completion-styles)) (add-to-list 'completion-styles (if (version< emacs-version "27") 'helm-flex 'flex) t) (call-interactively 'helm-M-x))
  spacemacs/helm-M-x-fuzzy-matching()
  funcall-interactively(spacemacs/helm-M-x-fuzzy-matching)
  call-interactively(spacemacs/helm-M-x-fuzzy-matching nil nil)
  command-execute(spacemacs/helm-M-x-fuzzy-matching)
```

**Comment by jkitchin** [comment]
that was a bug. See if the fix above addresses it. thanks for checking it out.

**Comment by fleimgruber** [comment]
Thanks for the quick fix, works for me with org-ref-20201231.1406.
